tests/puppeteer/campaigns/sanity/cartFO/01_editCheckCart.js

```diff
@@ -0,0 +1,76 @@
+// Using chai
+const {expect} = require('chai');
+
+// Importing pages
+const HomePage = require('../../../pages/FO/home');
+const ProductPage = require('../../../pages/FO/product');
+const CartPage = require('../../../pages/FO/cart');
+const CartData = require('../../data/FO/cart');
+
+let page;
+let homePage;
+let productPage;
+let cartPage;
+let totalTTC = 0;
+let itemsNumber = 0;
+
+// creating pages objects in a function
+const init = async () => {
+  page = await global.browser.newPage();
+  homePage = await (new HomePage(page));
+  productPage = await (new ProductPage(page));
+  cartPage = await (new CartPage(page));
+};
+
+/*
+  Open the FO home page
+  Add the first product to the cart
+  Add the second product to the cart
+  Check the cart
+  Edit the cart and check it
+ */
+global.scenario('Check the Product page', () => {
+  test('should open the shop page', async () => {
+    await homePage.goTo(global.URL_FO);
+    await homePage.checkHomePage();
+  });
+  test('should go to the first product page', () => homePage.goToProductPage('1'));
```

Comment:
```suggestion
  test('should go to the first product page', async () => await homePage.goToProductPage('1'));
```

And same for every non `async-await` method :)

Reply:
```js
  test('should go to the first product page', async () => {
await homePage.goToProductPage('1'));
});
```
For eslint

Reply:
It causes problems in the esLint :/